docker/cube/Dockerfile

```
@@ -1,5 +1,7 @@
 FROM cubejs/cube:v1.1.9
 
+RUN apt-get update && apt-get install -y curl
```

Just to keep with best practice for building images with minimal bloat

```suggestion
RUN apt-get update \
        && apt-get install -y --no-install-recommends curl \
        && apt-get clean \
        && rm -rf /var/lib/apt/lists/*
```